docs/hydra/cli/hydra-janitor.md

```diff
@@ -31,7 +31,7 @@ this command with caution.
 Janitor can be used in several ways.
 
 1. By passing the database connection string (DSN) as an argument
-   Pass the database url (dsn) as an argument to janitor. E.g. janitor <database-url>
+   Pass the database url (dsn) as an argument to janitor. E.g. janitor {database-url}
```

Comment:
This you will also need to fix in Ory Hydra itself :) Otherwise it will be overwritten on next push ...

Reply:
Right! I'll try to locate this file and create PR with fix in the CLI code. Thanks